### PR TITLE
Bugfix 6541/Fix Link clicks in tW and tA articles

### PR DESCRIPTION
**Pull request by PhotoNomad0**
#### Describe what your pull request addresses (Please include relevant issue numbers):
- fix problem with async loading of resources.

#### Please include detailed Test instructions for your pull request:
- can use test build attached below.
- make sure clicking links in tHelps pane work.  See https://github.com/unfoldingword/translationcore/issues/6541

### CODE REVIEW

**Comment by PhotoNomad0** [comment]
Test build:

From the `release-vx2.1.0-test.2` branch:
[Linux build 2.1.0 (fa6cd3b)](https://tc-builds.door43.org/build/release-vx2.1.0-test.2/translationCore-linux-x64-2.1.0-fa6cd3b.deb) is ready for download.
[macOS build 2.1.0 (fa6cd3b)](https://tc-builds.door43.org/build/release-vx2.1.0-test.2/translationCore-macos-x64-2.1.0-fa6cd3b.dmg) is ready for download.
[Windows x64 build 2.1.0 (fa6cd3b)](https://tc-builds.door43.org/build/release-vx2.1.0-test.2/translationCore-win-x64-2.1.0-fa6cd3b.setup.exe) is ready for download.
[Windows x32 build 2.1.0 (fa6cd3b)](https://tc-builds.door43.org/build/release-vx2.1.0-test.2/translationCore-win-x32-2.1.0-fa6cd3b.setup.exe) is ready for download.

File: src/components/TranslationHelpsWrapper.js [hunk 1]
```diff
@@ -51,26 +51,33 @@ function TranslationHelpsWrapper(props) {
   const groupId = contextId.groupId;
   const languageId = toolsSelectedGLs[currentToolName];
 
+  function getArticleFromReducer(resourceDir, article) {
```
Comment (mannycolon) [inline src/components/TranslationHelpsWrapper.js]: Would you mind adding the jsdocs documentation. Not sure what resourceDir is.

Comment (PhotoNomad0) [reply src/components/TranslationHelpsWrapper.js]: Good point.  Not sure I do either - will look it up.

Comment (mannycolon) [reply src/components/TranslationHelpsWrapper.js]: Okay thanks

**Pull request merged**